### ISSUE TITLE
Add cop name to offence output

### ISSUE THREAD

**Issue by lee-dohm** [rubocop/rubocop#102]
I ran into an issue today where some code was triggering a cop but upon reviewing the code I could find no preferable way to rewrite it, so I decided to disable that cop for just those few lines.  (Thanks @jonas054 for this fabulous feature!)  Unfortunately, I'm not as familiar with which cops do what ... so I had to go digging through the code to figure out which cop name to specify in the `rubocop:disable` comment.  This process should be easier.  I suggest that we change the output to something like the following:

``` text
C: 30: MethodLength: Method has too many lines. [12/10]
C: 86: ParameterLists: Avoid parameter lists longer than four parameters.
```
